rush.json

```diff
@@ -138,6 +138,12 @@
       ]
     },
     {
+      "packageName": "@microsoft/api-documenter",
+      "projectFolder": "libraries/api-documenter",
+      "reviewCategory": "libraries",
+      "shouldPublish": false  // not published; this is a code sample for api-extractor
+    },
```

Comment:
I'll do that later when there's some actual code there.

---
In reply to: [134340525](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134340525) [](ancestors = 134340525)